Sources/Sentry/Public/SentryEvent.h

```diff
@@ -76,7 +76,10 @@ NS_SWIFT_NAME(Event)
 @property (nonatomic, copy) NSString *_Nullable environment;
 
 /**
- * The current transaction (state) on the crash
+ * The current transaction (state) on the crash.
+ *
+ * This property is deprecated and will be removed in a future version of the SDK.
+ * Use SentryTransaction.transactionContext.name
```

Comment:
@bruno-garcia I think this make sense, we should start cleaning old stuff that dont make sense anymore, but I dont have all the background here.
Do you think we can remove this in the next major?

Reply:
Reverted